tests occasionally hang while executing locker_tests

tests occasionally hang while executing locker_tests on my raspberry-pi and the last time when testing on push to PR

![image](https://user-images.githubusercontent.com/35558969/62030516-0a5ea200-b207-11e9-9d50-adf123ae6de3.png)

Except for the case in the screenshot above, this only happened on my raspberry-pi. So before I 
 not believed that this problem is critical ( Because there is something I have with performance on raspberry ). Now that the problem has manifested itself not only there, I consider it necessary to localize and correct it.

```
F
tests/test_agent.py:886 (test_independent_lockers)
self = <module 'py.error'>, func = <built-in function open>
args = ('/tmp/pytest-of-root/pytest-0/test_independent_lockers0/test_locker_0', 'r')
kwargs = {}, __tracebackhide__ = False, cls = <class 'py.error.ENOENT'>
value = FileNotFoundError(2, 'No such file or directory')
tb = <traceback object at 0x714ef828>, errno = 2

    def checked_call(self, func, *args, **kwargs):
        """ call a function and raise an errno-exception if applicable. """
        __tracebackhide__ = True
        try:
>           return func(*args, **kwargs)
E           FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-root/pytest-0/test_independent_lockers0/test_locker_0'

../.wott-venv/lib/python3.5/site-packages/py/_error.py:66: FileNotFoundError

During handling of the above exception, another exception occurred:

tmpdir = local('/tmp/pytest-of-root/pytest-0/test_independent_lockers0')

    def test_independent_lockers(tmpdir):
>       one, two, both = _is_parallel(tmpdir, True, True)

tests/test_agent.py:888: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_agent.py:872: in _is_parallel
    return is_parallel(test_files), \
tests/test_agent.py:837: in is_parallel
    of = f.open('r')
../.wott-venv/lib/python3.5/site-packages/py/_path/local.py:366: in open
    return py.error.checked_call(open, self.strpath, mode)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <module 'py.error'>, func = <built-in function open>
args = ('/tmp/pytest-of-root/pytest-0/test_independent_lockers0/test_locker_0', 'r')
kwargs = {}, __tracebackhide__ = False, cls = <class 'py.error.ENOENT'>
value = FileNotFoundError(2, 'No such file or directory')
tb = <traceback object at 0x714ef828>, errno = 2

    def checked_call(self, func, *args, **kwargs):
        """ call a function and raise an errno-exception if applicable. """
        __tracebackhide__ = True
        try:
            return func(*args, **kwargs)
        except self.Error:
            raise
        except (OSError, EnvironmentError):
            cls, value, tb = sys.exc_info()
            if not hasattr(value, 'errno'):
                raise
            __tracebackhide__ = False
            errno = value.errno
            try:
                if not isinstance(value, WindowsError):
                    raise NameError
            except NameError:
                # we are not on Windows, or we got a proper OSError
                cls = self._geterrnoclass(errno)
            else:
                try:
                    cls = self._geterrnoclass(_winerrnomap[errno])
                except KeyError:
                    raise value
>           raise cls("%s%r" % (func.__name__, args))
E           py.error.ENOENT: [No such file or directory]: open('/tmp/pytest-of-root/pytest-0/test_independent_lockers0/test_locker_0', 'r')

../.wott-venv/lib/python3.5/site-packages/py/_error.py:86: ENOENT
```

--- a/tests/test_agent.py
+++ b/tests/test_agent.py
@@ -846,13 +846,13 @@ def _is_parallel(tmpdir, use_lock: bool, use_pairs: bool = False):
     # Schedule two identical tasks to executor. They will write before/after timestamps
     # to their files every 100 ms.
     test_files = [tmpdir / 'test_locker_' + str(i) for i in range(2)]
-    exes = [executor.Executor(0.1, sleeper, (use_lock, test_file, 'one')) for test_file in test_files]
+    exes = [executor.Executor(0.5, sleeper, (use_lock, test_file, 'one')) for test_file in test_files]
 
     # If testing independent locking, schedule another couple of tasks with another lock and another
     # set of timestamp files.
     if use_pairs:
         test_files_2 = [tmpdir / 'test_locker_2_' + str(i) for i in range(2)]
-        exes += [executor.Executor(0.1, sleeper, (use_lock, test_file, 'two')) for test_file in test_files_2]
+        exes += [executor.Executor(0.5, sleeper, (use_lock, test_file, 'two')) for test_file in test_files_2]
 
     futs = [executor.schedule(exe) for exe in exes]
 

